src/i18n/locale/en.json

```diff
@@ -369,21 +369,21 @@
     },
     "backup": {
       "label": {
-        "backup_restore_failed": "Backup restore failed!",
+        "backup_restore_failed": "Could not restore backup",
         "create_backup": "Create Backup",
-        "create_backup_info": "Backup library as a Tachiyomi backup",
+        "create_backup_info": "Back up library as a Tachiyomi backup",
         "legacy_backup_unsupported": "legacy backups are not supported!",
         "restore_backup": "Restore Backup",
-        "restore_backup_info": "You can also drag and drop the backup file here to restore",
-        "restored_backup": "Backup restore finished!",
-        "restoring_backup": "Restoring backup..."
+        "restore_backup_info": "You can also drag and drop the backup file here to restore it",
+        "restored_backup": "Backup restored.",
+        "restoring_backup": "Restoring backup…"
       },
       "title": "Backup"
     },
     "label": {
       "dark_theme": "Dark Theme",
       "image_cache": "Use image cache",
-      "image_cache_description": "Disabling image cache makes images load faster if you have a slow disk, but uses it much more internet traffic in turn",
+      "image_cache_description": "Preloading images shows them quicker, but downloads more data.",
```

Comment:
It was confusing as it was, now its worse

Reply:
I'm also unaware that the cache has something to do with preloading?

regarding the old description, I don't think any disk (unless broken) will actually have any impact.
so the only change I see, is that the images will always be freshly loaded from the source